components/org.wso2.carbon.identity.organization.management.application/src/main/java/org/wso2/carbon/identity/organization/management/application/OrgApplicationManagerImpl.java

```diff
@@ -582,6 +579,18 @@ private OAuthConsumerAppDTO createOAuthApplication(String mainAppName)
         return getOAuthAdminService().registerAndRetrieveOAuthApplicationData(consumerApp);
     }
 
+    private String resolveCallbackURL(String ownerOrgId) throws URLBuilderException, OrganizationManagementException {
+
+        String tenantDomain = getOrganizationManager().resolveTenantDomain(ownerOrgId);
+        ServiceURLBuilder commonAuthServiceUrl = ServiceURLBuilder.create().addPath(FrameworkConstants.COMMONAUTH)
+                .setTenant(tenantDomain);
+        if (!MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain) &&
+                isOrganizationQualifiedURLsSupported(ownerOrgId)) {
```

Comment:
```suggestion
                !isOrganizationQualifiedURLsDisabled(ownerOrgId)) {
```
We can refactor the method only if we are fine to resolve the suggestions for this PR.
https://github.com/wso2-extensions/identity-organization-management-core/pull/48